cargo-dist/src/host.rs

```diff
@@ -208,7 +212,21 @@ fn release_hosting(_dist: &DistGraph, manifest: &DistManifest, abyss: &Gazenot)
             None
         }
     });
-    tokio::runtime::Handle::current().block_on(abyss.create_releases(releases))?;
+
+    // Tell The Abyss To Release
+    let new_releases =
+        tokio::runtime::Handle::current().block_on(abyss.create_releases(releases))?;
+
+    // Update artifact download URLs with release results
+    for new_release in new_releases {
+        if let Some(new_url) = new_release.release_download_url {
+            manifest.update_release_axodotdev_artifact_download_url(&new_release.package, new_url);
+        }
+    }
+
+    // Update Github Announcement body with new URLs
+    announcement_github(manifest);
+
```

Comment:
Note to self: check if `should-publish` CI (the one that runs upload+release) actually reuploads the dist-manifest to scratch storage so that `announce` would see this update...